docs/stream-aggregation.md

```diff
@@ -65,6 +65,9 @@ Stream aggregation can be used as [statsd](https://github.com/statsd/statsd) alt
 * [Quantiles over input metrics](#quantiles-over-input-metrics)
 * [Histograms over input metrics](#histograms-over-input-metrics)
 
+Currently, streamin aggregation is available only for [supported data ingestion protocols](https://docs.victoriametrics.com/#how-to-import-time-series-data)
```

Comment:
@Amper looks like there is a typo `streamin aggregation` => `streaming aggregation`?